playbooks/installation/install_validation.adoc

```diff
@@ -109,16 +111,20 @@ Follow the steps below
 [Wait for build to start]
 # oc get pods | grep build
 [ copy the pod id ]
-# oc logs <pod id>
+# oc logs -f <pod id>
 [ Look for message saying an image was successfully pushed ]
-# oc get deployments
-[ make sure one has succeeded ]
+# oc get deploymentConfig
+[ copy the deploymentConfig name ]
+# oc deploy <deploymentConfig>
```

Comment:
@etsauer executing `oc deploy <deploymentConfig>` displays the latest deployment for the named DC.
```
$ oc deploy nexus
nexus deployment #2 deployed 10 hours ago - 1 pod
```
add --latest flag to start a new deployment, or --retry flag retry the latest failed deployment, or the --cancel flag to cancel an in progress deployment 

All else looks good to me.